src/Adapter/MSTest.CoreAdapter/ObjectModel/UnitTestElement.cs

```diff
@@ -229,25 +229,26 @@ internal TestCase ToTestCase()
             {
             }
 
-            var idProvider = new TestIdProvider();
-            idProvider.AppendString(testCase.ExecutorUri?.ToString());
-            idProvider.AppendString(fileName);
-            if (this.TestMethod.HasManagedMethodAndTypeProperties)
-            {
-                idProvider.AppendString(this.TestMethod.ManagedTypeName);
-                idProvider.AppendString(this.TestMethod.ManagedMethodName);
-            }
-            else
-            {
-                idProvider.AppendString(testCase.FullyQualifiedName);
-            }
-
-            if (this.TestMethod.DataType != DynamicDataType.None)
-            {
-                idProvider.AppendString(testCase.DisplayName);
-            }
-
-            testCase.Id = idProvider.GetId();
+            // This code is disabled for compatibility. 
```

Comment:
Would it be better to remove this code instead of commenting it out ? We'll always have the git log should we wish to restore it.

Reply:
I almost sure that change will make people mad, so the commenting. If we're not asked to revert this change after preview - I'll remove the code.